bindings/ros/aditof_roscpp/src/aditof_utils.cpp

```diff
@@ -27,7 +28,14 @@ std::shared_ptr<Camera> initCameraEthernet(int argc, char **argv) {
     }
 
     std::vector<std::shared_ptr<Camera>> cameras;
-    system.getCameraListAtIp(cameras, ip);
+    if (argc < 2) {
```

Comment:
Why not check if ip is not empty?

Reply:
Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/250/commits/9f03a7a6556856aa50f986fbbbf5b0ceffd0dab8